server/src/main/java/com/tisza/tarock/game/card/Card.java

```diff
@@ -11,6 +11,8 @@
 	public abstract int getPoints();
 	public abstract String getID();
 	public abstract boolean isHonor();
+
+	public abstract int getValue();
```

Comment:
Although getValue is present in both subclasses, it doesn't makes sense to make it an abstract method here, as SuitCard.getValue and TarockCard.getValue have quite different meanings.

If you want to sort cards for bots, I would suggest writing a sorting function that uses casts to compare Card objects.